anti-analysis/anti-vm/vm-detection/check-for-sandbox-username-hostname.yml

```diff
@@ -15,7 +16,9 @@ rule:
       - ccbf7cba35bab56563c0fbe4237fdc41:0x402B90
   features:
     - and:
-      - api: GetUserName
+      - or:
+        - api: GetUserName
```

Comment:
substitute match for https://github.com/mandiant/capa-rules/blob/master/host-interaction/session/get-session-user-name.yml?